docs/usage/examples/opentelemetry.md

```diff
@@ -0,0 +1,115 @@
+# OpenTelemetry
+
+Requirements:
+
+- docker-compose
+
+## Prepare setup
+
+Create a `docker-compose.yaml` and `otel-collector-config.yml` file as seen below in a folder
+
+`docker-compose.yaml`:
+
+```yaml
+version: '3'
+services:
+  # Jaeger
+  jaeger:
+    image: jaegertracing/all-in-one:latest
```

Comment:
I'm slightly torn on this file: There are a significant amount of docker image references on this page, for our own and for third party products. Some of the references contain explicit versions, others refer to "latest". It's inconsistent, and since this file is in an "examples" folder, it will not be updated by Renovate, like we also see for https://github.com/renovatebot/.github/issues/31.

Reply:
Yeah, best to pin to `:1` in case they make breaking changes?